bio/jellyfish/count/environment.yaml

```
@@ -0,0 +1,6 @@
+channels:
+  - bioconda
+  - conda-forge
+  - defaults
+dependencies:
+  - kmer-jellyfish==2.3.0
```

I there a particular reason for this to be pinned to `2.3.0`? Otherwise I'd opt for:
```suggestion
  - kmer-jellyfish==2.3
```

That way, bug fixes and especially stuff like security patches will automatically be used whenever available.